features/FEATURE_LWIP/lwip-interface/EthernetInterface.cpp

```diff
@@ -19,10 +19,15 @@
 
 
 /* Interface implementation */
-EthernetInterface::EthernetInterface()
-    : _dhcp(true), _ip_address(), _netmask(), _gateway()
+EthernetInterface::EthernetInterface() : 
+    _dhcp(true), 
+    _ip_address(), 
+    _netmask(), 
+    _gateway()
 {
 }
+Callback<void(nsapi_connection_status_t, int)> EthernetInterface::_connection_status_cb = NULL;
```

Comment:
These things mustn't be static. Nothing stopping you having multiple Ethernet interfaces.